src/test/java/core/basesyntax/file/impl/FileReaderImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.file.impl;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.FileProcessingException;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.file.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class FileReaderImplTest {
```

Comment:
place all files in the test/resources directory. Move paths to the constant fields